components/form/TextField.styled.ts

```diff
@@ -11,7 +11,11 @@ const StyledTextField = styled.TextInput`
   padding-bottom: 6px;
   color: #ffffff;
   border-bottom-width: 1px;
-  border-bottom-color: #ffffff;
+  border-bottom-color: ${props => (props.valid ? '#fff' : '#f00')};
```

Comment:
I would propose to have an error color in our theme/colors module.

Reply:
But should we do it even if it is just white or black? 
I agree if it comes to error color, I will put it in theme colors module

Reply:
I'm not sure 😄 Not really, but maybe we would decide to choose `#f5f5f5` as `white`, and then it might make some sense 😄